activators/lex/README.md

```diff
@@ -0,0 +1,96 @@
+<p style="text-align: center">
+    <img src="https://www.routeoneconnect.com/wp-content/uploads/2020/03/icon-amazon-lex.png" height="128" width="128" alt="Lex logo"/>
+</p>
+
+<h1 style="text-align: center">Lex NLU activator</h1>
+
+Allows using [Amazon Lex](https://aws.amazon.com/lex/) NLU engine as a states activator in JAICF.
+
+_Built on top of [Amazon Lex API Client for Java](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/)._
+
+## How Lex works
+
+You can read how Lex works [here](https://docs.aws.amazon.com/lex/latest/dg/how-it-works.html)
+
+## How to use
+
+#### 1. Include Lex dependency to your _build.gradle_
+
+```kotlin
+implementation("com.just-ai.jaicf:lex:$jaicfVersion")
+```
+
+**Replace `$jaicfVersion` with the latest
+version ![](https://img.shields.io/github/v/release/just-ai/jaicf-kotlin?color=%23000&label=&style=flat-square)**
+
+#### 2. Use Lex `activator` in your scenario actions
+
+```kotlin
+state("launch") {
+    activators {
+        intent("BookHotel")
+    }
+
+    action(lex) {
+        // Recognised named entities
+        val slots = activator.slots
+
+        // Data of recognized intent 
+        val recognizedIntent = activator.recognizedIntentData
+    }
+}
+```
+
+#### 3. Sign up in the AWS Management Console
+
+To create an AWS account, open [Aws Management sign up page](https://portal.aws.amazon.com/billing/signup) and follow
+the online instructions.
+
+Part of the sign-up procedure involves receiving a phone call and entering a verification code on the phone keypad.
+
+#### 4. Sign in Lex console
+
+Open [Lex Console](https://console.aws.amazon.com/lexv2/home) and sign in. Select the second version of the console.
+
+#### 5. Create a bot
+
+You can create a bot using this [blueprint](https://docs.aws.amazon.com/lex/latest/dg/ex-book-trip.html) or create your
```

Comment:
This link points to a lot of text about bot, but does not create one. We need a better link, like 
https://console.aws.amazon.com/lexv2/home#createBot which actually creates bot.